.bcr/config.yaml

```diff
@@ -0,0 +1 @@
+fixedReleaser: alexeagle
```

Comment:
The schema for this changed. `fixedReleaser` now takes `login` and `email` fields.

Reply:
Also, I think the `.yml` extension is assumed. I should make that more flexible..